subworkflows/local/bam_variant_calling_tumor_only_all/main.nf

```diff
@@ -60,6 +60,7 @@ workflow BAM_VARIANT_CALLING_TUMOR_ONLY_ALL {
     // CONTROLFREEC (depends on MPILEUP)
     if (tools.split(',').contains('controlfreec')) {
         length_file = cf_chrom_len ?: fasta_fai
+        ploidy = params.cf_ploidy && !params.cf_ploidy.toString().contains(",") ? [params.cf_ploidy] : params.cf_ploidy.split(',')
```

Comment:
Shouldn't we have that in the upstream subworkflows and feed this one ploidy?

Reply:
yes agreed.  I am still trying around what will work. Some of the scripts apparently can deal with the list, some don't. It's a bit confusing.